src/docker/README.md

```diff
@@ -9,20 +9,26 @@ build the processor, otherwise you will get errors.
 git submodule update --init --recursive
 ```
 
-Ensure that your environment variables are set, typically with an `.env`
```

Comment:
- [ ] Merge blocking dependency cascade beginning with ECO-2695 before this merges, so that we are not sharing out instructions for a stack that will ultimately fail if run against mainnet (because the current processor in the hanging release needs to get resolved)